tools/c7n_azure/c7n_azure/policy.py

```diff
@@ -85,7 +85,7 @@ class AzureFunctionMode(ServerlessExecutionMode):
 
     POLICY_METRICS = ('ResourceCount', 'ResourceTime', 'ActionTime')
 
-    default_storage_name = "cloudcustodian"
+    default_storage_name = "custodian"
```

Comment:
Update docs to reflect this change?

Reply:
Silly storage account names max at 24 characters....